pkg/virt-controller/services/template_test.go

```diff
@@ -1688,6 +1694,27 @@ var _ = Describe("Template", func() {
 
 				Expect(pod.Spec.Affinity).To(BeNil())
 			})
+
+			It("should add affinity to pod of vmi host model", func() {
+				config, kvInformer, svc = configFactory(defaultArch)
+				vmi := v1.VirtualMachineInstance{
+					ObjectMeta: metav1.ObjectMeta{Name: "testvm", Namespace: "default", UID: "1234"},
+					Spec: v1.VirtualMachineInstanceSpec{
+						Domain: v1.DomainSpec{
+							Devices: v1.Devices{
+								DisableHotplug: true,
+							},
+							CPU: &v1.CPU{
+								Model: "host-model",
+							},
+						},
+					},
+				}
+				pod, err := svc.RenderLaunchManifest(&vmi)
+				Expect(err).ToNot(HaveOccurred())
+
+				Expect(pod.Spec.Affinity).ToNot(BeNil())
```

Comment:
Isn't more robust to expect the exact value?

Reply:
Right, Thanks!
Done.